### PR TITLE
Finish docs

### PR DESCRIPTION
**Pull request by pyzon**
Finished documenting the subtasks of the project.

### CODE REVIEW

File: doc/ManualTests.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 ===
 
-//TODO: feladat leírása
+Manuális tesztek megtervezése, végrehajtása és dokumentálása
 
-## Megoldás
+## Tervezés
 
-//TODO: Mintaszöveg
+A tervezés során létrehoztam egy példa person.xml-t és person.xsd-t a resources/manual/person mappában. Ezeket adom át egy pár soros minta kódnak, amit elvégzi az átalakítást. A kimenetet a Protoge programban nyitom meg, itt olvasható módon megjelenik, hogy sikeres volt-e a konverzió.
```
Comment (boben98) [inline doc/ManualTests.md:8]: Typo: "kódnak, ami**t**"
```suggestion
A tervezés során létrehoztam egy példa person.xml-t és person.xsd-t a resources/manual/person mappában. Ezeket adom át egy pár soros minta kódnak, ami elvégzi az átalakítást. A kimenetet a Protoge programban nyitom meg, itt olvasható módon megjelenik, hogy sikeres volt-e a konverzió.
```

File: doc/connectSonarCloud.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Connect SonarCloud
+===
+
+A SonarCloud statikus kódellenőrző összekötése a GitHub projekttel, hogy ezután a Sonar ellenőrző is lefusson push-olás után.
```
Comment (boben98) [inline doc/connectSonarCloud.md:4]:
```suggestion
A SonarCloud statikus kódellenőrző összekötése a GitHub projekttel, hogy ezután a Sonar ellenőrző is lefusson push-olás és pull request után.
```

File: doc/connectSonarCloud.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Megoldás
+
+A SonarCloud weboldalán megnyitottuk a GitHub projektet, és beállítottuk a manuális kódellenőrzést. A `pom.xml` fájlba beleíródtak a SonarCloud beállításai:
```
Comment (boben98) [inline doc/connectSonarCloud.md:8]:
```suggestion
A SonarCloud weboldalán megnyitottuk a GitHub projektet, és beállítottuk a manuális kódellenőrzést. A `pom.xml` fájlba beleírtuk a SonarCloud beállításait:
```

File: doc/coverage.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Coverage with JaCoCo
+===
+
+A lefedettség mérésére kellett egy eszközt integrálni a projektbe.
+A választás a laborgyakorlaton is használt JaCoCo-ra esett.
+
+## Megoldás
+
+Ahhoz, hogy részletes lefedettségi információt kapjunk, elég volt a függőségek közé
+hozzáadni a JaCoCo-t pom.xml-ben, illetve egy maven plugint deklarálni a `<plugins>` szekcióban.
```
Comment (boben98) [inline doc/coverage.md:10]:
```suggestion
felvenni a JaCoCo-t `pom.xml`-ben, illetve egy maven plugint deklarálni a `<plugins>` szekcióban.
```

File: doc/coverage.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+Ez html formátumban legenerálta számunkra a statisztikát.
```
Comment (boben98) [inline doc/coverage.md:11]:
```suggestion
Ez több formátumban legenerálta számunkra a statisztikát, amelyek közül az XML-t a SonarCloud is fel tudja használni.
```

File: doc/connectSonarCloud.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+A sonar tokent a GitHub-on secretként tároltuk el, és a workflow-t (`maven.yml`) frissítettük ennek megfelelően:
+
+```yml
+- name: Build with Maven
+    run: mvn -B package sonar:sonar --file pom.xml
+    env:
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+```
+
```
Comment (boben98) [inline doc/connectSonarCloud.md:28]:
```suggestion

Az automata futás sokáig nem működött, viszont többszöri manuálisan hívott ellenőrzés és kisebb javítások után életre kelt.

```

File: doc/setupBuildAndCI.md [hunk 7]
```diff
@@ -1,18 +1,34 @@
 Setup CI
 ===
 
-//TODO: feladat leírása
+Fordítás automatizálás és folytonos integráció megvalósítása Maven valamint GitHub Actions segítségével.
```
Comment (boben98) [inline doc/setupBuildAndCI.md:4]:
```suggestion
Fordítás automatizálás és folytonos integráció megvalósítása Maven, valamint GitHub Actions segítségével.
```

File: doc/connectSonarCloud.md (hunk 2 above)
Comment (pyzon) [reply doc/connectSonarCloud.md]: Jogos

**Pull request merged**